build/docker/intel-idxd-config-initcontainer.Dockerfile

```diff
@@ -59,6 +59,9 @@ COPY --from=builder /accel-config.tar.gz /licenses/accel-config/
 
 ADD demo/idxd-init.sh /idxd-init/
 ADD demo/dsa.conf /idxd-init/
+ADD demo/iaa.conf /idxd-init/
+
+RUN mkdir /idxd-init/scratch
```

Comment:
not related to this change as such but I'd propose we move `idxd-init.sh ` to `/usr/local/bin` (see #1000) at some point. can we make this scratch space so that it's not relative to the script to avoid rework.

Reply:
@mythi 
idxd-init.sh -> /usr/local/bin/